docs/a320-simvars.md

```diff
@@ -1400,23 +1400,28 @@
         - 2
         - 3
 
-- A32NX_PAX_TOTAL_ROWS_{rows}
-    - Number
-    - Indicates the current number of pax in the selected rows
-    - {rows}
-        - 1_6
-        - 7_13
-        - 14_21
-        - 22_29
+- A32NX_BOARDING_STARTED_BY_USR
+    - Bool
+    - Indicates current pax/cargo loading state
 
-- A32NX_PAX_TOTAL_ROWS_{rows}_DESIRED
-    - Number
-    - Indicates the target number of pax in the selected rows
+- A32NX_PAX_FLAGS_{station}
+    - Bitwise
+    - Indicates the current pax in the selected rows (max 53 bits)
```

Comment:
You can't safely write 53-bits to a simvar from JS. A single precision float is used in the backend somewhere.

Reply:
resolved on discord, turns out the limit is indeed 53-bits :P